pom.xml

```diff
@@ -703,6 +703,11 @@
         <artifactId>mockito-core</artifactId>
         <version>${mockito.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-inline</artifactId>
+        <version>${mockito.version}</version>
+      </dependency>
```

Comment:
Adding `mockito-inline` module to support final method mocking. See [here](https://davidvlijmincx.com/posts/mockito_mock_final_methods_and_classes/).

Reply:
I think that's fine.